### PR TITLE
[JENKINS-65161] remove commons-digester2 from core and upgrade plugins to commons-digester3

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

**Comment by olamy** [comment]
https://github.com/jenkinsci/jenkins/pull/5320

File: com.ibm.team.build.hjplugin/pom.xml [hunk 1]
```diff
@@ -99,12 +99,28 @@
 		</repository>
 	</repositories>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>commons-beanutils</groupId>
+				<artifactId>commons-beanutils</artifactId>
+				<!-- 1.9.4 when this merged https://github.com/stapler/stapler/pull/211 -->
```
Comment (oleg-nenashev) [inline com.ibm.team.build.hjplugin/pom.xml:107]: Copy-paste comment? It is not that trivial for a plugin, a core version would need to be updated

**Review by oleg-nenashev — APPROVED** [review]
Should there be a beanutils-api plugin?

**Comment by olamy** [comment]
sounds weird at the end we will turn to have a plugin for each shared library...

File: com.ibm.team.build.hjplugin/pom.xml (hunk 1 above)
Comment (olamy) [reply com.ibm.team.build.hjplugin/pom.xml:107]: just enforce it's the good version for digester3 we have here.

Comment (olamy) [reply com.ibm.team.build.hjplugin/pom.xml:107]: actually digester3 only need 1.8.3 but I try to have something higher to avoid security scan trigger

**Review by daniel-beck — CHANGES_REQUESTED** [review]
Likely creates a security vulnerability as described in https://github.com/jenkinsci/jenkins/pull/5320#issuecomment-809979530

**Comment by olamy** [comment]
@daniel-beck fixed

**Review by rsandell — APPROVED** [review]

**Comment by olamy** [comment]
@scowan @lvaikunt ping as you are the main maintainers

**Review by alecharp — APPROVED** [review]

**Review by rsandell — APPROVED** [review]

**Review by MRamonLeon — APPROVED** [review]

**Comment by alecharp** [comment]
@lvaikunt do you think you could merge this PR and cut a release of the plugin? Thanks

**Review by lvaikunt — COMMENTED** [review]
Does this require an update to the core Jenkins version or parent pom?

**Comment by lvaikunt** [comment]
@alecharp Sure, I have started the review.

**Comment by MRamonLeon** [comment]
> Does this require an update to the core Jenkins version or parent pom?

No. It's just preparing the plugin for when the dependency is removed from Core.

**Comment by olamy** [comment]
@lvaikunt ping :)

**Comment by basil** [comment]
@lvaikunt FYI, jenkinsci/jenkins#5320 has been approved by @jenkinsci/core-pr-reviewers and @jenkinsci/core and will likely be merged and released soon. To avoid impacting users of this plugin, I suggest this PR be merged and released at the earliest convenience.

**Review by lvaikunt — APPROVED** [review]

**Comment by lvaikunt** [comment]
Hello all,
Sorry for the delay. Thanks for the info in the previous comments. I will merge this change.

Regarding a release of the plugin, I think we can schedule one soon after performing the required testing.

**Comment by batmat** [comment]
FYU @lvaikunt the Jenkins Core PR got merged. 

This means **starting next Monday _teamconcert_ will fail to load** on the latest weekly if this PR does not get a release.

**Comment by lvaikunt** [comment]
A new version of Team concert plugin (2.4.0) with the above changes will be released today.

**Comment by lvaikunt** [comment]
Team Concert Plugin 2.4.0 released on June 21 2021.

**Pull request closed without merging**